src/main/scala/com/maxmind/gatling/dev/MockServer.scala

```diff
@@ -0,0 +1,104 @@
+package com.maxmind.gatling.dev
```

Comment:
Normally yes, but here I moved mockserver and the test libs into production side.

Reasons:

1) Unlike most projects, production side is a test project. So some testing utilities from specs2 for example, are useful.

2) Very useful as you are developing gatling scripts, changing config, etc. to open the mock server and get back a log of what your simulation simulated, and it is useful to do so even when inside the mm_website dir for example, or from somewhere you do not have the project checkout dir. In such cases all you have is the all-in-one project jar.

Could have done this by creating two jars, main.jar and test.jar, which seems to work using the SBT Assembly plugin, or by adding some tests into main jar using include filters, which I could not get to work. So instead we add test dependencies to production, and move classes we need in production + test into the "main" area, and self-tests not useful in production to the "tests" area of the src tree.